### DISCUSSION TITLE
Will there be an "official" Gatsby generator for integrated NX Monorepos?

### DISCUSSION THREAD

**Discussion by Megajin** [gatsbyjs/gatsby#37277] [Ideas / Feature Requests]
Hi everyone,

I am struggling with a integrated monorepo using nxjs. There was a generator to create easily Gatsby apps called `@nrwl/gatsby` https://www.npmjs.com/package/@nrwl/gatsby . As you can see, the package is deprecated. The versions aren't up to date either and I did run into issues with some of the other react libs in that monorepo. I know that I could use a package-based monorepo. But that would bring up other issues like linking all the dependencies manually...

Which brings me to my question/feature-request:

Is there any possibility for an "official" gatsby plugin for nx monorepos to generate apps (especially for integrated repos)?

I am asking this, because in the Gatsby documentation is a note saying: Gatsby "fully" supports nx monorepos --https://www.gatsbyjs.com/docs/reference/cloud/monorepos/ . Which brought up that idea of a plugin which generates all needed things and places executors with the newest versions of Gatsby.

**Comment by trentschnee** [comment]
I second this
